integrations/docker/images/chip-build-tizen/tizen-sdk-installer/install.sh

```diff
@@ -251,20 +254,33 @@ function install_tizen_sdk() {
     ln -sf ../../lib/libcap.so.2 "$TIZEN_SDK_SYSROOT/usr/lib/libcap.so"
     ln -sf openssl1.1.pc "$TIZEN_SDK_SYSROOT/usr/lib/pkgconfig/openssl.pc"
 
+    # Setup ccache links
+    info "Setting up ccache links..."
+    CCACHE=$(command -v ccache || echo "/usr/bin/ccache")
+    mkdir -p "$TIZEN_SDK_TOOLCHAIN_CCACHE/bin"
+    ln -sf "$TIZEN_SDK_TOOLCHAIN/bin/arm-linux-gnueabi-ar" "$TIZEN_SDK_TOOLCHAIN_CCACHE/bin"
```

Comment:
Is this required? Note that GN supports `pw_command_launcher` argument that you can set to `ccache`.

Reply:
Hmm I was not aware of the `pw_command_launcher` option. It seems that it works out of the box. Thx!

If it will for with other targets as well, maybe it should be added to `build_examples.py` as an option flag?